addon/components/nrg-modal/component.js

```diff
@@ -33,6 +33,7 @@ export default Component.extend({
   notMasterDetail: not('masterDetail'),
   notLightbox: not('lightbox'),
   notSidebar: not('sidebar'),
+  notRenderInPlace: not('renderInPlace'),
```

Comment:
Does [this](https://github.com/knoxville-utilities-board/ember-nrg-ui/blob/2d0cd183dfc85a55cea28037394d738b2e22f540/addon/components/nrg-modal/component.js#L30) line not do the same thing? I think you just need to change the variable referenced on L24 to use `renderInModal` instead.

Reply:
Indeed it does.